### ISSUE TITLE
Increase Minimum Pool Payouts

### ISSUE THREAD

**Issue by MoneroUKJack** [SChernykh/p2pool#172]
In the case a miner chooses regular payouts at the minimum 0.0003 XMR currently around $0.07 and decides at a later point to make a transaction with their mining reward of $70 when this user consolidates these inputs to spend they could be creating a transaction that could consist of 1,000 inputs. Given the RingCT ring size increase to 16 (July22) this could mean a total of upto 16,000 would need to be included in the transaction causing unnecessary large transactions.

Is there any consideration to increase the minimum payout to a higher amount at a later date?

**Comment by jameswillhoite** [comment]
There is no "pool wallet" payouts are immediately when a block is found and you have 1+ shares in the window. Basically this is solo mining with some help.

**Comment by SChernykh** [comment]
There's no "minimum payout". Miners get paid per share when they have a share in PPLNS window and a block is mined.

**Comment by MoneroUKJack** [comment]
In an event of P2pool reaching any larger % of the hash rate as more miners migrate over  would this create even smaller payouts to more users creating more “tiny” inputs when a miner consolidates or sends funds? 

I’m asking a miner currently a size of his transaction made recently combining all of his payouts/shares so this can be looked at. 

Is there a potential for a feature that could hold the first 100 shares and then consolidate into 1 payout or is this out scope or negates that benefits of p2pool? 

When something is the default users are less likely to opt out where as if users are advised to consolidate these over times into usable outputs for future spends they are highly unlikely to do this then selves. 

Any thoughts or somewhere better to discuss this?

**Comment by SChernykh** [comment]
No, this will not create smaller payouts. The smallest payout is fixed at 1/2160 of the block reward.

> 100 shares and then consolidate into 1 payout

I ran simulations when I developed p2pool, and this scheme underpays small miners. Proper accounting requires keeping the whole p2pool sidechain from block 1 which is not an option (too slow syncing of the new nodes). P2Pool sidechain is transient, only the latest 4320 blocks exist at any moment.
